homeassistant/components/eps/const.py

```diff
@@ -0,0 +1,22 @@
+"""Constants for the iAlarm integration."""
+from pyepsalarm import EPS
+
+from homeassistant.const import (
+    STATE_ALARM_ARMED_AWAY,
+    STATE_ALARM_ARMED_NIGHT,
+    STATE_ALARM_DISARMED,
+    STATE_ALARM_TRIGGERED,
+)
+
+DATA_COORDINATOR = "eps"
```

Comment:
This doesn't seem to be used, please remove it.

Reply:
Thanks !